### PR TITLE
Create intermediate build against libpq=13

### PR DESCRIPTION
**Pull request by akrherz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
*  Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

conda-forge currently has some dependency ugliness as a result of the `qt` feedstock being in a state of unbuildable flux.  I have some lengthy thrashing denoted at conda-forge/gdal-feedstock#585 

The long story short is that we have a very narrow dependency path that presently includes gdal=3.4.0 + libpq 13.  This creates an intermediate build of pdal against libpq=13, which would hopefully allow PRs like conda-forge/qgis-feedstock#218 to work.

If accepted, I'll submit a subsequent PR that undoes this pinning.

### CODE REVIEW

**Comment by hobu** [comment]
@akrherz what prompted the need for this build? We try really hard not to pin anything so that all of the updates from the rest of the CondaForge team make it into the builds without interruption.

**Comment by akrherz** [comment]
Hi @hobu, I hope you are well :)

It was prompted by conda-forge/qgis-feedstock#218 and this:

```
$ mamba create -n test python=3.10 qt=5.12.9 pdal=2.3.0 libgdal=3.4.0
- package qt-5.12.9-h1f2b2cb_0 requires icu >=67.1,<68.0a0
```

That's a terse error, but I am rather certain it is coming from what I stated above.  This has been a painful two weeks attempting to get this all working given what is happening with `qt` now :(

**Comment by hobu** [comment]
The issue is probably QT, of course, but that's a heavy lift to upgrade. QGIS is going to get EoL'd here in CondaForge because of lack of QT 6 in one or two more releases I believe.

**Comment by akrherz** [comment]
There's a lot of work on-going within QT conda-forge, but no valid builds since February and the present master branch doesn't rebuild as is without any changes, so we are stuck with things like `icu=68`.  I have had some success with lots of ugliness like this to get various repos to build.  I believe there are folks that are going to make qgis work again once the QT work is done. I dunno.

**Comment by akrherz** [comment]
Oh thanks for the leap of faith @hobu, reverting PR coming shortly...

**Comment by hobu** [comment]
It'll take like 2 hours before the packages are posted.

**Comment by ocefpaf** [comment]
Sorry I was too late but @akrherz approach here is fine until we fix qt and maybe add the libpqd pin to the central pinning (I thought we already had that :facepalm:).

**Comment by akrherz** [comment]
Thanks @ocefpaf , the conda-forge/conda-forge-pinning-feedstock#133 was the suggested pinning of libpq

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf , the [conda-forge/conda-forge-pinning-feedstock#133](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/133) was the suggested pinning of libpq

That is probably correct. I also don't really want to go that route :-)
